test/rules/prefer-conditional-expression/check-else-if/test.ts.lint

```diff
@@ -51,4 +51,56 @@ if (true) {
     foo(bar).baz = 1;
 }
 
+if (length === 0) {
+~~ [err % ('x')]
+    x = "foo";
+} else if (length === 1) {
+    x = "bar";
+} else if (length === 2) {
+    x = "something else";
+} else {
+    x = "unknown";
+}
+
+if (length === 0) {
+~~ [err % ('x')]
+    x = "foo";
+} else {
+    if (length === 1) {
+        x = "bar";
+    } else {
+        if (length === 2) {
```

Comment:
The error used to be reported on this line. I fixed that while I was already touching the code.